### PR TITLE
Added option to context menu to copy filepath

### PR DESCRIPTION
**Pull request by torbei**
shortcut for it is ctrl-shift-c

Changed test for modifer keys to work correctly.
More info about the issue here:
https://developer.gnome.org/gtk3/stable/checklist-modifiers.html

Should fix issue #74

### CODE REVIEW

File: src/clipboard.c [hunk 1]
```diff
@@ -117,3 +117,19 @@ clipboard_copy_file_list (GList* file_list, guint32 copy)
...
+    for (GList *file = file_list; file != NULL; file = file->next)
+    {
+        gchar* file_name = NULL;
+        file_name = (char*) file->data;
+
+        g_string_append(filepathlist , file_name);
+        g_string_append(filepathlist , "\r\n");
+    }
+    gtk_clipboard_set_text (clip, filepathlist->str, filepathlist->len);
+}
```
Comment (cboxdoerfer) [inline src/clipboard.c]: There's a memory leak. `filepathlist` isn't freed.j

File: src/fsearch_window_actions.c [hunk 2]
```diff
@@ -273,6 +273,21 @@ fsearch_window_action_copy (GSimpleAction *action,
...
+                                     GVariant      *variant,
+                                     gpointer       user_data)
+{
+    FsearchApplicationWindow *self = user_data;
+    GtkTreeSelection *selection = fsearch_application_window_get_listview_selection (self);
+    if (selection) {
+        GList *file_list = NULL;
+        gtk_tree_selection_selected_foreach (selection, copy_file, &file_list);
+        file_list = g_list_reverse (file_list);
+        clipboard_copy_filepath_list (file_list);
```
Comment (cboxdoerfer) [inline src/fsearch_window_actions.c]: That's an implicit declaration. `clipboard_copy_filepath_list` isn't exported in `clipboard.h`.

**Comment by torbei** [comment]
Thanks for the help. It's my first time doing something with GTK.
I pushed a new commit, which should fix it

**Comment by cboxdoerfer** [comment]
Yes, it looks fine now. Once I'm home I'll build the code and if everything works we can merge it.

**Comment by cboxdoerfer** [comment]
Works as expected 👍

**Pull request merged**